Entities/ACrab.cpp

```diff
@@ -2357,37 +2377,28 @@ void ACrab::Update()
     ////////////////////////////////////
     // Aiming
 
-	//float rotAngle = GetRotAngle();
+    // Get rotation angle of crab
+    float RotAngle = GetRotAngle();
```

Comment:
RotAngle should be rotAngle
Looks like in the codebase variable names are pascalCase (i.e. start lower case, upper case for start of every word) while methods and classes and stuff are CamelCase (i.e. upper case for start of every word)